app/controllers/api/v2/users.rb

```diff
@@ -3,7 +3,7 @@ module V2
     class Users < API::Base
       include API::V2::Defaults
 
-      resource :users do
+      resource :users, desc: "Deprecated" do
```

Comment:
A request to `"/api/v2/swagger_doc"` responds with the following under the "api" key:

on master
```
[{"path"=>"/bikes_search", "description"=>"Searching for bikes"},
 {"path"=>"/bikes", "description"=>"Operations about bikes"},
 {"path"=>"/me", "description"=>"Operations about the current user"},
 {"path"=>"/manufacturers", "description"=>"Accepted manufacturers"},
 {"path"=>"/selections", "description"=>"Selections (static options)"}]
```

on this branch
```
[{"path"=>"/bikes_search", "description"=>"Searching for bikes"},
 {"path"=>"/bikes", "description"=>"Operations about bikes"},
 {"path"=>"/me", "description"=>"Operations about the current user"},
 {"path"=>"/users", "description"=>"Operations about users"},
 {"path"=>"/manufacturers", "description"=>"Accepted manufacturers"},
 {"path"=>"/selections", "description"=>"Selections (static options)"}]
```

(The Users API is mounted but all routes within are marked as hidden.)

Reply:
This is acceptable.